### ISSUE TITLE
the option --env prevent the environment GOPROXY from being set to more than one proxy

### ISSUE THREAD

**Issue by xgfone** [fyne-io/fyne-cross#35]
`GOPROXY` supports more than one proxy, which are separated by the comma. But the option `--env` also requires that the arguments must be separated by the comma. They are not compatible with each other.

```shell
$ fyne-cross windows -env GOPROXY='https://goproxy.io,direct'
invalid value "GOPROXY=https://goproxy.io,direct" for flag -env: env var must defined as KEY=VALUE or KEY=
```

After removing `,direct`, it works.

**Question:** How will I set the environment `GOPROXY` to many proxies by the option `--env` or others.

**Comment by lucor** [comment]
Good catch. This should be allowed and will be fixed in the upcoming releases.

In the meantime, as workaround, you could "extend" the docker image like:

```
cat <<EOF > Dockerfile
FROM fyneio/fyne-cross:base-latest
ENV GOPROXY=https://goproxy.io,direct
EOF

docker build -t  fyne-cross-with-proxy .

fyne-cross windows --image fyne-cross-with-proxy
```

**Comment by lucor** [comment]
Available in develop for testing via #36
